Source/CBLRemoteRequest.m

```diff
@@ -176,7 +176,9 @@ - (NSMutableDictionary*) statusInfo {
 
 - (void) respondWithResult: (id)result error: (NSError*)error {
     Assert(result || error);
-    _onCompletion(result, error);
+
+    CBLRemoteRequestCompletionBlock onCompletion = _onCompletion;   // keep block alive till return
```

Comment:
Should this also use `__typeof(_onCompletion) onCompletion = _onCompletion ` like the others?